src/applications/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -3,11 +3,12 @@ import _ from 'lodash';
 import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
 
 import dateUI from '../../../common/schemaform/definitions/date';
-import SSNWidget from '../../../common/schemaform/widgets/SSNWidget';
+import PhoneNumberWidget from '../../../common/schemaform/widgets/PhoneNumberWidget';
```

Comment:
There is an additional option for implementing the phone field using the common uiSchema and a BE transformation listed in this issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/10897

Reply:
Um...Is that the ticket you meant to link?

Reply:
Yep, although it's an unrelated issue, it may be of interest here in the event that we want to be more consistent with the other forms (which seem to use the common phone uiSchema -- which itself uses the `PhoneNumberWidget`), I think I set this up using the widget before we decided to start using the BE to perform transformations. Not sure what the preference is here so mentioning it in case it's helpful.

Reply:
Ohhhh, I gotcha. You weren't referencing the problem in the ticket, but the proposed solution therein. :ok_hand:
I'm on the same page now. Thanks!